indigo_app/views/works.py

```diff
@@ -405,18 +406,34 @@ def get_permission_required(self):
             return ('indigo_api.delete_commencement',)
         return ('indigo_api.view_commencement',)
 
+    def get(self, request, *args, **kwargs):
+        if request.htmx.target == 'commencement-new':
+            # delete the new commencement and reload the page
+            work_commencements_url = self.get_work_commencements_url()
+            self.do_delete()
```

Comment:
it's dangerous to do a deletion operation on a GET call

Reply:
My understanding is we do this when we're "creating" a new commencement and want to cancel. Since "creating a new commencement" is actually "create and edit", we need to delete if we cancel. Perhaps we change the experience of "create" to "create and edit" explicitly, so then "cancel" becomes "delete".

Linked to this: what's the experience if I click "new commencement" and then refresh the page -- presumably an empty commencement? What's the experience if I then click "edit"?